danuluma/andela-project
# #160702239 Ft authentication

#### What does this PR do?
Adds authentication using jwt

#### Description of Task to be completed?
adds the endpoints:
```~/dann/api/v1/reg``` --> here a new user can register to the app in order to get access to secured endpoints
```~/dann/api/v1/login``` --> a returning user logs in using his/her username and password to get access to secured endpoints

#### How should this be manually tested?
* Clone the repo.
* Run ```python3 run.py```
* Fire up postman and navigate to POST ```/dann/api/v1/reg```. Register to get an access token.
* Create another request to GET ```/dann/api/``` . It should return unauthorized access.
* Now add to the request header the access token gotten previously in the POST ```/dann/api/v1/reg```. It should return "Hello world". You will have successfully gained access using an authorization token.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#160702239
#### Screenshots (if appropriate)
#### Questions:
